mitmproxy/tools/console/flowlist.py

```diff
@@ -57,6 +57,10 @@ def __init__(self, master):
         self.master = master
         urwid.ListBox.__init__(self, master.logbuffer)
 
+    def set_focus(self, index):
+        if(0 <= index < len(self.master.logbuffer)):
```

Comment:
Please remove the `(...)` around the if condition - we don't use this style of code in mitmproxy.